release/rllib_tests/app_config.yaml

```diff
@@ -13,7 +12,3 @@ post_build_cmds:
   - sudo rm -rf /home/ray/anaconda3/lib/python3.7/site-packages/numpy
   - pip3 install numpy==1.19.5 || true
```

Comment:
Can we figure out a way to remove line 12 and 13 please?

Reply:
Yes, I was going to raise this in the product repo. This does not come up in the Docker container at runtime, only during build of cluster envs, so this is likely an issue with caching.

Reply:
why can't we bake this into the docker files?

Reply:
This is one of the pain points for our existing users. It won't work unless we fix this to not require post build commands.
If you think about how a customer sees this, then they will think that ray-ml is all they need.

Reply:
It makes sense. I'll merge this PR first and will follow up with the product team. I don't think this has to do with the `rayproject/ray-ml` docker image, but with the `anyscale/` one